OpenChange/iCalEvent+MAPIStore.m

```diff
@@ -372,6 +372,7 @@ - (void) updateFromMAPIProperties: (NSDictionary *) properties
   if (value)
     {
       NSArray *recipients;
+      int ccFirstIndex;
```

Comment:
If we have "int i" to iterate in the for, using the same type for another index is what I understand as coherency, or are you proposing also to change the type of "i" to NSUInteger?